packages/docker.rb

```diff
@@ -0,0 +1,161 @@
+# Adapted from Arch Linux docker PKGBUILD at:
+# https://github.com/archlinux/svntogit-community/raw/packages/docker/trunk/PKGBUILD
+
+require 'package'
+
+class Docker < Package
+  description 'Pack, ship and run any application as a lightweight container'
+  homepage 'https://www.docker.com/'
+  version '23.0.0-rc.1'
+  license 'Apache'
+  compatibility 'all'
+  source_url 'https://github.com/docker/cli.git'
+  git_hashtag "v#{version}"
+
+  binary_url({
+    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/docker/23.0.0-rc.1_armv7l/docker-23.0.0-rc.1-chromeos-armv7l.tar.zst',
+     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/docker/23.0.0-rc.1_armv7l/docker-23.0.0-rc.1-chromeos-armv7l.tar.zst',
+     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/docker/23.0.0-rc.1_x86_64/docker-23.0.0-rc.1-chromeos-x86_64.tar.zst'
+  })
+  binary_sha256({
+    aarch64: '7cd26fca47393832c15012ce0345f78c688b1062e1fb3eef985db625cd1be9cd',
+     armv7l: '7cd26fca47393832c15012ce0345f78c688b1062e1fb3eef985db625cd1be9cd',
+     x86_64: 'afc88b554da28cce205a42de3a7a9abc0d567fd08b5d0fd66c77f95d728368c4'
+  })
+
+  depends_on 'bridge_utils'
+  depends_on 'btrfsprogs' => :build
+  depends_on 'containerd'
+  depends_on 'elogind' => :build
+  depends_on 'eudev'
+  depends_on 'glibc' # R
+  depends_on 'go' => :build
+  depends_on 'go_md2man' => :build
+  depends_on 'iproute2'
+  depends_on 'lvm2' # R
+  depends_on 'sqlite'
+
+  no_env_options
+  no_fhs
+
+  def self.build
+    @cli_version = git_hashtag
+    @moby_version = git_hashtag
+    @libnetwork_version = '1f3b98be6833a93f254aa0f765ff55d407dfdd69'
+    @tini_version = '378bbbc8909a960e89de220b1a4e50781233a740'
+    @buildx_version = '8a7a221a7fd636bafd6bf05b964f8b87b65d8877'
```

Comment:
Should these be separate packages?

Reply:
Not how arch is doing these packages...  https://github.com/archlinux/svntogit-community/raw/packages/docker/trunk/PKGBUILD

These are just repos for the dependent libraries, which won't be used otherwise...

Reply:
They can be pulled out as separate packages later if necessary.